uk.co.compucorp.civicrm.tasksassignments/views/crm/case-type-category-form-field.tpl

```diff
@@ -0,0 +1,11 @@
+<div crm-ui-field="{literal}{ name: 'caseTypeDetailForm.category', title: ts('Category') }{/literal}">
```

Comment:
The views folder is mainly used for angular templates. Can we add this to the templates folder instead.